tasklist/client/src/Tasks/Task/Details/TabListNav.tsx

```diff
@@ -0,0 +1,62 @@
+ * Condition 1: If the Licensee distributes the Software or any derivative works of the Software, the Licensee must attach this Agreement.
+ * Condition 2: Without limiting other conditions in this Agreement, the grant of rights is solely for non-production use as defined below.
+ * "Non-production use" means any use of the Software that is not directly related to creating products, services, or systems that generate revenue or other direct or indirect economic benefits.  Examples of permitted non-production use include personal use, educational use, research, and development. Examples of prohibited production use include, without limitation, use for commercial, for-profit, or publicly accessible systems or use for commercial or revenue-generating purposes.
+ *
+ * If the Licensee is in breach of the Conditions, this Agreement, including the rights granted under it, will automatically terminate with immediate effect.
+ *
+ * SUBJECT AS SET OUT BELOW, THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ * NOTHING IN THIS AGREEMENT EXCLUDES OR RESTRICTS A PARTY’S LIABILITY FOR (A) DEATH OR PERSONAL INJURY CAUSED BY THAT PARTY’S NEGLIGENCE, (B) FRAUD, OR (C) ANY OTHER LIABILITY TO THE EXTENT THAT IT CANNOT BE LAWFULLY EXCLUDED OR RESTRICTED.
+ */
+
+import {useNavigate} from 'react-router-dom';
+import styles from './TabListNav.module.scss';
+import cn from 'classnames';
+
+type Props = {
+  className?: string;
+  label: string;
+  items: Array<{
+    key: string;
+    title: string;
+    label: string;
+    selected: boolean;
+    href: string;
+  }>;
+};
+
+const TabListNav: React.FC<Props> = ({className, label, items}) => {
+  const navigate = useNavigate();
+  return (
+    <nav className={cn(className, styles.tabs, 'cds--tabs')}>
+      <div className="cds--tab--list" aria-label={label}>
+        {items.map(({key, title, label, selected, href}) => (
+          <button
+            key={key}
+            type="button"
+            role="link"
+            aria-label={label}
+            aria-current={selected ? 'page' : undefined}
+            className={cn('cds--tabs__nav-item', 'cds--tabs__nav-link', {
+              ['cds--tabs__nav-item--selected']: selected,
+            })}
+            tabIndex={selected ? 0 : -1}
+            onClick={() => navigate(href)}
+          >
+            <div className="cds--tabs__nav-item-label-wrapper">
+              <span className="cds--tabs__nav-item-label">{title}</span>
+            </div>
+          </button>
+        ))}
+      </div>
+    </nav>
```

Comment:
@douglasbouttell-camunda why are we rebuilding this component instead of using the built-in one?

Reply:
Because the Tabs component isn't compatible with being based around a router and lazy loading. It uses contexts to set up a bunch of aria properties so it knows what tabs it controls. Doing it with the existing component causes a11y to fail.